datasette/utils/__init__.py

```diff
@@ -70,6 +70,30 @@ async def await_me_maybe(value):
     return value
 
 
+def convert_list_to_sql(list, descending=False):
+    """Takes a Python list, escapes its values and returns it as comma separated values"""
+    value_list = ""
+    for item in list:
+        value_list += f"{escape_sqlite(item)}{' desc' if descending else ''}, "
+    return value_list[:-2]
```

Comment:
This should be a list that you'd `join()` with ", " as a separator when returning to get a string
```suggestion
    values = [escape_sqlite(item) + ' desc' if descending else ''
              for item in list]
    return values.join(", ")
```